lib/notifiers/notifiers_test.go

```diff
@@ -29,13 +29,28 @@ import (
 	"testing"
 	"time"
 
+	"github.com/golang/protobuf/ptypes"
+
 	"github.com/golang/protobuf/proto"
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
 	cbpb "google.golang.org/genproto/googleapis/devtools/cloudbuild/v1"
 	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
+func convertToTimestamp(datetime string) *timestamppb.Timestamp {
+	timestamp, err := time.Parse(time.RFC3339, datetime)
+	if err != nil {
+		panic(err)
```

Comment:
In general, we try to not use panic in lieu of error handling (see [here](https://golang.org/doc/effective_go.html#panic)). Can you pass in t *testing.T as a param and call t.Fatalf instead of panic here?